BE/src/main/java/kr/codesquad/issuetracker/domain/UserDTO.java

```diff
@@ -0,0 +1,32 @@
+package kr.codesquad.issuetracker.domain;
+
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Map;
+
+@Getter
+@ToString
+public class UserDTO {
+
+    private final String nickname;
+    private final String email;
+
+    private UserDTO(String nickname, String email) {
+        this.nickname = nickname;
+        this.email = email;
+    }
+
+    private UserDTO(Map<String, String> userMap) {
+        this.nickname = userMap.get("nickname");
+        this.email = userMap.get("email");
+    }
+
+    public static UserDTO of(String nickname, String email) {
+        return new UserDTO(nickname, email);
+    }
+
+    public static UserDTO of(Map<String, String> userMap) {
+        return new UserDTO(userMap);
+    }
```

Comment:
```suggestion

    public static UserDTO of(String nickname, String email) {
        return new UserDTO(nickname, email);
    }

    public static UserDTO of(Map<String, String> userMap) {
        return new UserDTO(userMap.get("nickname"), userMap.get("email"));
    }

    public static UserDTO of(User user) {
        return new UserDTO(user.getNickname(), user.getEmail());
    }
```

그리고 User 객체 자체를 넘겨받아 DTO를 만들어주는 메소드도 필요하다고 생각합니다~
좀 더 객체지향적인 접근을 하는 것이 좋을 듯 해요!

Reply:
이거는 위에부분 수정하면서 만들었습니당 ㅋㅋ